pipelines/datasets/us_bls_qcew/constants.py

```
@@ -81,3 +81,30 @@ class constants(Enum):
+    # ── Recurring pipeline (NAICS only) ──────────────────────────────────────
+    # SIC is a frozen classification: BLS published it for 1975-2000 and never
+    # republishes it. The recurring pipeline therefore refreshes ONLY the 8
+    # NAICS data tables; the 8 SIC tables and the dicionario are static and are
+    # not in the pipeline's table loop.
+    NAICS_START_YEAR = 1990
+
+    NAICS_TABLES = [
+        f"naics_{f}_{g}"
+        for f in ["quarterly", "annual"]
+        for g in ["national", "state", "county", "metro"]
+    ]
```

Ruff (0.16.0)

[warning] Mutable default value for class attribute

(RUF012)